test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -60,4 +65,17 @@ public static Person generateTestPerson() {
             return null;
         }
     }
+    
+    public static void compareFiles(Path path1, Path path2) throws IOException {
```

Comment:
`isSameTextIgnoreLineEndings(): boolean`? Just return a boolean instead of coupling this file to JUnit.
Add a header comment.